bentoml/_internal/bento/bento.py

```diff
@@ -321,19 +324,32 @@ def creation_time(self) -> datetime:
 
     @inject
     def save(
-        self, bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store]
+        self,
+        bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
+        model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ) -> "Bento":
         self.flush_info()
 
         if not self.validate():
             logger.warning(f"Failed to create Bento for {self.tag}, not saving.")
-            raise BentoMLException("Failed to save Bento because it was invalid")
+            raise BentoMLException("Failed to save Bento because it was invalid.")
 
         with bento_store.register(self.tag) as bento_path:
             out_fs = fs.open_fs(bento_path, create=True, writeable=True)
             fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
             self._fs.close()
             self.__fs = out_fs
+            track(
+                event_properties=BentoBuildEvent(
+                    bento_tag=self.info.tag,
+                    bento_creation_timestamp=self.info.creation_time,
+                    bento_size_in_kb=calc_dir_size(bento_path),
```

Comment:
add a `models_size_in_kb` here? with the dir size of `os.path.join(bento_path, "models")`

Reply:
done